### PR TITLE
Consolidate deps into NPM.

### PR DESCRIPTION
**Pull request by donmccurdy**
Proposing to remove `bower.json` (A-Frame isn't available on Bower anyway) and consolidate deps into `package.json`. This has the nice side effect of making the version of ThreeJS explicit.

I think this requires TypeScript v2+, FYI.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,5 +18,8 @@
     "gulp-typescript": "^2.12.2",
     "gulp-uglify": "^1.5.3",
     "jshint-stylish": "^2.1.0"
+  },
+  "dependencies": {
```
Comment (xirvr) [inline package.json]: Minor thing, but would be cleaner to put this in dev deps or move everything to regular deps (either is fine with me... just wouldn't wanna come back to this some day and wonder why the difference ;)

Comment (donmccurdy) [inline package.json]: Oops, yep not sure what I was thinking.

**Comment by donmccurdy** [comment]
Moved everything to devDependencies + rebased.

(still planning to look into a PR for A-Frame core but i haven't gotten to that yet...)

**Comment by xirvr** [comment]
Looks good, merging.

**Pull request merged**